package.json

```diff
@@ -47,6 +47,7 @@
     "react-dom": "^18.0.0",
     "shx": "^0.3.4",
     "start-server-and-test": "^1.14.0",
-    "typescript": "^4.5.2"
+    "typescript": "^4.5.2",
+    "victory-vendor": "^36.6.3"
```

Comment:
Is this package used at this level? We also have it in the Nuka package

Reply:
No it's not, I'll remove that. Nice catch!